sdk/src/store.rs

```diff
@@ -3604,6 +3677,10 @@ pub enum InvalidClaimError {
     #[error("the verifiable credentials store could not be read")]
     VerifiableCredentialStoreInvalid,
 
+    /// The feature is not supported by version
+    #[error("the manifest contained a feature not support by version")]
+    VersionFailure,
```

Comment:
Consider naming this something like `UnsupportedVersion` or `UnsupportedFeature`. If it's about features, maybe a string describing what feature isn't supported.

Reply:
done